code/modules/mining/satchel_ore_boxdm.dm

```diff
@@ -16,13 +16,20 @@
 		W.loc = src
 	if (istype(W, /obj/item/weapon/storage))
 		var/obj/item/weapon/storage/S = W
-		S.hide_from(usr)
 		for(var/obj/item/weapon/ore/O in S.contents)
 			S.remove_from_storage(O, src) //This will move the item to this item's contents
-		user << "<span class='notice'>You empty the satchel into the box.</span>"
+		user << "<span class='notice'>You empty the ore in [S] into the [src].</span>"

```

Comment:
so change the line to this?
`user << "<span class='notice'>You empty the ore in [S] into \the [src].</span>"`